Drop-In/src/test/java/com/braintreepayments/api/AddCardActivityUnitTest.java

```diff
@@ -965,6 +965,46 @@ public void showsSubmitButtonAgainWhenThreeDSecureIsCanceled() throws JSONExcept
         assertThat(mEditCardView.findViewById(R.id.bt_button)).isVisible();
     }
 
+    @Test
+    public void addCard_onSuccess_returnsToApp() {
```

Comment:
Are we waiting to refactor these tests when `AddCardActivity` has been converted to a fragment? If so, can we add a TODO?

Reply:
So just realized these tests do belong in DropIn, the name threw me off. Going to move these back into `DropInActivityUnitTest`.